lib/endpoints/discover.js

```diff
@@ -104,6 +104,17 @@ function apiGetContainers(req, res, next) {
                     payload.containers = containers.filter(
                         function inGroup(container) {
                             var groups = container.groups;
+                            /*
+                             * If this is poseidon, append manta groups  to the
```

Comment:
Super nitpick: there are two spaces between "groups" and "to" on this line. Also, I'd prefer if "manta groups" were changed to "the manta group" because there's only ever one manta group.

Reply:
Ah, thanks for catching that!